content/en/user-guide/tools/cloud-pods/community/index.md

```diff
@@ -1,10 +1,14 @@
 ---
 title: "Community Cloud Pods"
-weight: 4
+weight: 99
 description: >
   Get started with LocalStack Community Cloud Pods to to save and load your container state at will
 ---
 
+{{< alert title="Note" >}}
+With the release of LocalStack 2.3, we have officially marked Community Cloud Pods as **deprecated**. In our upcoming 3.0 release, we will be removing Community Cloud Pods entirely. Moving forward, Cloud Pods will only be accessible to Pro/Team users. We understand this may affect some users and we are open to feedback, through our [GitHub Issues](https://github.com/localstack/localstack) or [Discuss Forum](https://discuss.localstack.cloud/).
```

Comment:
should we mention the insider program for users to keep using community pods?

Reply:
Nope, external programs that we run shouldn't be mentioned on docs which totally serve a technical purpose to general purpose users